src/table/components/__tests__/TableBodyWrapper.spec.tsx

```diff
@@ -38,49 +55,58 @@ describe('<TableBodyWrapper />', () => {
           keyboard={keyboard}
           tableWrapperRef={tableWrapperRef}
           announce={announce}
-        />
+        >
+          {children}
+        </TableBodyWrapper>
       </TableContextProvider>
     );
 
   beforeEach(async () => {
     jest.spyOn(selectionsUtils, 'addSelectionListeners').mockImplementation(() => jest.fn());
-    tableData = await manageData(model, generateLayout(1, 1, 2, [], [{ qText: '100' }]), { top: 0, height: 100 });
+    tableData = (await manageData(
+      model,
+      generateLayout(1, 1, 2, [], [{ qText: '100' }]),
+      { top: 0, height: 100 } as unknown as PageInfo,
+      () => undefined
+    )) as TableData;
     constraints = {};
     selectionsAPI = {
       isModal: () => true,
-    };
+    } as ExtendedSelectionAPI;
     theme = {
-      getColorPickerColor: () => {},
-      name: () => {},
-      getStyle: () => {},
+      getColorPickerColor: () => undefined,
+      name: () => undefined,
+      getStyle: () => undefined,
       table: { body: { borderColor: '' } },
-    };
-    layout = {};
+    } as unknown as ExtendedTheme;
+    layout = {} as TableLayout;
     cellRendererSpy = jest.fn();
+    tableFirstRow = tableData.rows[0]['col-0'] as Cell;
   });
 
   afterEach(() => jest.clearAllMocks());
 
   it('should render 2x2 table body and call CellRenderer', () => {
     jest.spyOn(getCellRenderer, 'default').mockImplementation(() => {
       cellRendererSpy();
-      return 'td';
+      return 'td' as unknown as CellHOC;
```

Comment:
I think this is way more complicated than needed. Try just `jest.spyOn(getCellRenderer, 'default')` and then `    expect(getCellRenderer.default).toHaveBeenCalledTimes(2);`

Reply:
Goes for the other tests below as well